datajoint/settings.py

```diff
@@ -45,7 +45,7 @@
     'display.width': 14,
     'display.show_tuple_count': True,
     'database.use_tls': None,
-    'enable_python_native_blobs': False,  # python-native/dj0 encoding support
+    'enable_python_native_blobs': True,  # python-native/dj0 encoding support
```

Comment:
^ thinking this needs an adjustment to the README.md (maybe moving the old notes somewhere or removing but leaving some URL for the deleted text)

Reply:
Agreed. Will update before release.